client/public/locales/en/common.json

```diff
@@ -158,6 +160,8 @@
             "used_weight": "How much filament has been used from the spool. A new spool should have 0g used.",
             "remaining_weight": "How much filament is left on the spool. For a new spool this should match the spool weight.",
             "measured_weight": "How much the filament and spool weigh.",
+            "initial_weight": "The initial Weight of the filament (net weight)",
```

Comment:
Perhaps expand on this a bit as to why you would want to have this different from the filament setting. Justify to the user why adding this field was necessary.